skgrid/tools/skgrid-mlib.cwl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/skgrid/mlib.sh"]
+hints:
+  DockerRequirement:
+    dockerPull: "skgrid"
+
+inputs:
+  cancer:
+    type: string
+    inputBinding:
+      position: 1
+  platform:
+    type: string
+    inputBinding:
+      position: 2
+
+outputs:
+  mlib_out:
+    doc: tbd
```

Comment:
Fill it in or delete it

Reply:
rm in commit [6d9cf87](https://github.com/ohsu-comp-bio/gdan-tmp-models/pull/7/commits/6d9cf874e004ebd7cb926d000d4e98a77fafaa98)